lib/internal/Magento/Framework/Shell/CommandRenderer.php

```diff
@@ -16,8 +16,12 @@ class CommandRenderer implements CommandRendererInterface
      */
     public function render($command, array $arguments = [])
     {
-        $arguments = array_map('escapeshellarg', $arguments);
+        do {
+            $command = preg_replace('/\s?2>&1\s?\|/', ' |', $command, -1, $count);
+        } while ($count > 0);
```

Comment:
You don't need a cycle to replace arbitrary amount of occurrences: https://regex101.com/r/kgZCj5/1